examples/own_dataset/train.py

```diff
@@ -50,17 +53,37 @@ def __call__(self, atoms, adjs):
         return x
 
 
+class ScaledAbsError(object):
+
+    def __init__(self, scaler=None):
+        self.scaler = scaler
+
+    def __call__(self, x0, x1):
+        if isinstance(x0, Variable):
+            x0 = cuda.to_cpu(x0.data)
+        if isinstance(x1, Variable):
+            x1 = cuda.to_cpu(x1.data)
+        if self.scaler is not None:
+            scaled_x0 = self.scaler.inverse_transform(cuda.to_cpu(x0))
+            scaled_x1 = self.scaler.inverse_transform(cuda.to_cpu(x1))
+            diff = scaled_x0 - scaled_x1
+        else:
+            diff = cuda.to_cpu(x0) - cuda.to_cpu(x1)
+        return numpy.mean(numpy.absolute(diff), axis=0)[0]
+
+
 def main():
     # Supported preprocessing/network list
     method_list = ['nfp', 'ggnn', 'schnet', 'weavenet', 'rsgcn']
     scale_list = ['standardize', 'none']
 
     parser = argparse.ArgumentParser(
         description='Regression with own dataset.')
-    parser.add_argument('datafile', type=str)
+    parser.add_argument('--datafile', type=str, default='dataset.csv')
```

Comment:
This breaks the example test scripts. Because `datafile` is not mandatory, a superfluous argument is given. Here is an error message:

```
%(pr-171)[U]bash -x example_test_cpu.sh 
...
+ python train.py dataset.csv --method nfp --label value1 --conv-layers 1 --gpu -1 --epoch 1 --unit-num 10 --batchsize 32
usage: train.py [-h] [--datafile DATAFILE]
                [--method {nfp,ggnn,schnet,weavenet,rsgcn}]
                [--label LABEL [LABEL ...]] [--scale {standardize,none}]
                [--conv-layers CONV_LAYERS] [--batchsize BATCHSIZE]
                [--gpu GPU] [--out OUT] [--epoch EPOCH] [--unit-num UNIT_NUM]
                [--seed SEED] [--train-data-ratio TRAIN_DATA_RATIO]
                [--protocol PROTOCOL]
train.py: error: unrecognized arguments: dataset.csv
```
IMHO, we do not have to set the default value as long as usage is written in `README.md` at the cost of breaking existing user code which invokes this script. How do you think?